providers/fips/self_test.c

```diff
@@ -469,20 +454,13 @@ void ossl_set_error_state(const char *type)
 
 int ossl_prov_is_running(void)
 {
-    int res;
-    static unsigned int rate_limit = 0;
+    int res, loclstate;
+    static TSAN_QUALIFIER unsigned int rate_limit = 0;
```

Comment:
Do we care about this rate_limit?

Reply:
I think we care enough about getting some messages to include this.  Without any thread safety, it's possible for zero error messages to appear which would be bad.

It's on an non-critical error path, so the cost isn't relevant IMO.